mail/getmail6/Portfile

```diff
@@ -0,0 +1,50 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+PortGroup           github 1.0
+
+github.setup        getmail6 getmail6 6.19.03 v
+
+revision            0
+categories          mail python
+platforms           {darwin any}
+maintainers         wolfsden.cz:~+macports
+license             GPL-2
+description         extensible mail retrieval system with POP3, IMAP4, SSL support
+long_description    getmail6 is a flexible, extensible mail retrieval          \
+                    system with support for POP3, IMAP4, SSL variants of both, \
+                    maildirs, mboxrd files, external MDAs, arbitrary message   \
+                    filtering, single-user and domain-mailboxes, and many      \
+                    other useful features.
+# getmail does not contain binary code
+supported_archs     noarch
+
+conflicts           getmail
+
+homepage            https://getmail6.org
+
+checksums           rmd160  992edce4932d14b4fdd1ef84fa04c8040a6bad5d \
+                    sha256  b46a257bd221b7ee0e55f4f8e84e7a2fee8c05ec3b61c08298c3605df250a14a \
+                    size    215166
+
+patchfiles          0001-pop3-Support-use_xoauth2.patch
+patch.pre_args      -p1
+
+# Not sure how far back to go.  3.5 is probably a safe guess.
+python.versions         312 311 310 39 38 37 36 35
```

Comment:
this doesn't do anything if the portname is not "py-<name>" so please remove. If one would support different Python versions we would certainly only add the ones that are not EOL.